ui/app/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -49,22 +56,24 @@ export default class ConfirmApproveContent extends Component {
           'confirm-approve-content__card--no-border': noBorder,
         })}
       >
-        <div className="confirm-approve-content__card-header">
-          <div className="confirm-approve-content__card-header__symbol">
-            {symbol}
-          </div>
-          <div className="confirm-approve-content__card-header__title">
-            {title}
-          </div>
-          {showEdit && (
-            <div
-              className="confirm-approve-content__small-blue-text cursor-pointer"
-              onClick={() => onEditClick()}
-            >
-              Edit
+        {showHeader && (
+          <div className="confirm-approve-content__card-header">
+            <div className="confirm-approve-content__card-header__symbol">
+              {symbol}
             </div>
-          )}
-        </div>
+            <div className="confirm-approve-content__card-header__title">
+              {title}
+            </div>
+            {showEdit && (
+              <div
+                className="confirm-approve-content__small-blue-text cursor-pointer"
+                onClick={() => onEditClick()}
```

Comment:
[REQ] if no additional arguments are required for these it's safe, and might save a render, to just pass the method directly. E.G `onClick={onEditClick}`

Reply:
This is an existing condition, will it be wise to change it as a part of this PR or in a new PR?!

Reply:
Ah i didn't catch that this was a relocated file! You can ignore my comments